src/Common/CurrentDirectoryChange.cs

```diff
@@ -0,0 +1,26 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using System.IO;
+
+namespace Microsoft.Oryx.Common
+{
+    public class CurrentDirectoryChange : IDisposable
+    {
+        private readonly string _originalCurrentDirectory;
+
+        public CurrentDirectoryChange(string newCurrentDirectory)
```

Comment:
I like this idea. What about removing the `Change` from the name, so we can call `using new CurrentDirectory("abc")`?

Reply:
I decided to remove this, based on Kiran's comment, and pass the source path to Oryx as an argument - just like in the `build` command.